### PR TITLE
Workflow Automated Test - LRQA-68830 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -775,12 +775,17 @@ definition {
...
 	test ShowsMessageWhenSearchReturnsNoResults {
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
 
-		// TODO LRQA-68830 ShowsMessageWhenSearchReturnsNoResults pending implementation
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
 
+		Workflow.searchAssignedToMyRoles(searchTerm = "12345");
+
+		AssertTextEquals(
+			locator1 = "//div[contains(@class,'sheet-text text-center')]",
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:787]: You can use [ locator1 = "Message#EMPTY_INFO" ].

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Review by lucasperj — APPROVED** [review]

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/Workflow.macro [hunk 2]
```diff
@@ -816,6 +816,14 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	macro searchAssignedToMyRoles {
+		Type(
+			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_INPUT",
+			value1 = "${searchTerm}");
+
+		Click(locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_ICON");
+	}
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/Workflow.macro:825]: Change the macro name to catch both areas (AssignedToMe/AssignedToMyRoles)

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path [hunk 3]
```diff
@@ -113,6 +113,16 @@
...
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_INPUT</td>
+	<td>//input[contains(@name,'keywords')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_ICON</td>
+	<td>//button[contains(@class, 'btn btn-monospaced btn-unstyled')][2]</td>
+	<td></td>
+</tr>
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path:125]: You can make the name of those paths more clear, you can use that same path to both areas (Assigned to me / Assigned to my roles)
Suggestion:

MY_WORKFLOW_TASKS_SEARCH_INPUT_FIELD
MY_WORKFLOW_TASKS_SEARCH_ICON
Remember to change the names at the created macro (searchAssignedToMe)

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Comment by PauloVAS** [comment]
@lucasperj , fixed!

**Comment by PauloVAS** [comment]
ci:test:sf

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
Just started reviewing :)

**Comment by lucasperj** [comment]
Combined and sent on [PR#3](https://github.com/lucasperj/liferay-portal/pull/3). Thank you!

**Pull request closed without merging**